### ISSUE TITLE
Unable to read xrdml file

### ISSUE THREAD

**Issue by raziel2701** [paruch-group/xrdtools#3]
I get the following:

```
xrdtools.io.read_xrdml('test.xrdml')
  File "<string>", line unknown
XMLSyntaxError: Start tag expected, '<' not found, line 1, column 1
```

It seems like the xrdml file is not correctly formatted but I've tested several files including one from your repo: test_scan.xrdml and it still won't work. This was tried on windows 7.

I tried regular one dimensional scans (2theta-omega scan) as well as a two axis scan with the same result.

**Comment by bziegler** [comment]
This is strange indeed. Which version of python and lxml are you using? I tested with python 2.7.11 and lxml 3.4.4 and 3.5.0 on a OS X and my test files are loaded correctly.

**Comment by GoMrPickles** [comment]
Hi, I also get this error. I get it using Python 3.6, but NOT when using Python 2.7. Is there a known Python 3 issue? I'm happy to work on a fix, but my knowledge of LXML is limited. SO question: https://stackoverflow.com/questions/49122406/lxml-issue-parsing-xrdml-schema-in-python-3

**Comment by GoMrPickles** [comment]
I made this change to line 43 of `io.py`: `with open(schema, 'r', encoding='utf8') as f:`, which got everything running smoothly in Python 3.

**Comment by bziegler** [comment]
Thanks for the solution. I implemented it and it's tested with python 2.7, 3.5 and 3.6 in the  xrdtools-dev-0.1.1 branch. Feel free to test it.
